### ISSUE TITLE
Test many MVs in e2e test

### ISSUE THREAD

**Issue by kwannoel** [risingwavelabs/risingwave#14143]
1. Many MV on MV deeply nested.
2. Many flat MV on the same table.

**Comment by xxchan** [comment]
Isn't this covered by longevity test? 🤔

**Comment by kwannoel** [comment]
> Isn't this covered by longevity test? 🤔

Yeah but longevity test takes a long time to debug. If it exists as a e2e test, we can quickly check the scenario and easily reproduce it.
